apps/src/code-studio/pd/professional_learning/courses/SelfPacedPLCatalogCourseFacilitatedWorkshops.tsx

```diff
@@ -0,0 +1,157 @@
+import {LinkButton} from '@code-dot-org/component-library/button';
+import Image from '@code-dot-org/component-library/image';
+import Link from '@code-dot-org/component-library/link';
+import Tags from '@code-dot-org/component-library/tags';
+import {
+  BodyThreeText,
+  BodyFourText,
+  StrongText,
+} from '@code-dot-org/component-library/typography';
+import React from 'react';
+
+import {CourseOfferingFacilitatedWorkshop} from '@cdo/apps/templates/courseOfferings/types';
+import findMoreWorkshopsIllustration from '@cdo/static/professional-learning/courses/find-more-workshops-illustration.svg';
+import workshopsTeachWithConfidenceIllustration from '@cdo/static/professional-learning/courses/workshops-teach-with-confidence-illustration.svg';
+
+import moduleStyles from './selfPacedPLCatalog.module.scss';
+
+type SelfPacedPLCatalogCourseFacilitatedWorkshopsProps = {
+  facilitated_workshops: CourseOfferingFacilitatedWorkshop[];
+};
+
+function formatDateString(date: string): string {
+  const d = new Date(date);
+  return d.toLocaleDateString('en-US', {
+    year: '2-digit',
+    month: '2-digit',
+    day: '2-digit',
+  });
+}
+
+function formatWorkshopDate(sessions: {start: string}[]): string {
+  if (!sessions.length) return '';
+  const formatted = formatDateString(sessions[0].start);
+
+  return sessions.length > 1
+    ? `${formatted} +${sessions.length - 1}`
```

Comment:
Nit: Could there be a space after the plus so it's "[date] + 4" rather than "[date] +4" for example.